### ISSUE TITLE
File > Export Model is grayed out for all *.gr2 files. Unable to export to 3ds or other formats.

### ISSUE THREAD

**Issue by fishscene** [ray/triexporter#4]
File > Export Model is grayed out for all *.gr2 files. Saving the *.gr2 file appears to work, but I can't find anything that can open/extract/convert.

![2019-03-27 16_10_17-Window](https://user-images.githubusercontent.com/1910351/55118776-b737df80-50ac-11e9-825e-7bee2216acbe.png)

**Comment by AxarTiberius** [comment]
I'm also having this issue. Tried the granny2 file at https://forums.eveonline.com/t/triexporter-still-works/59230/2 but still unable to export. Also, double clicking on a model causes TriExporter to crash.

**Comment by Vizis** [comment]
> I'm also having this issue. Tried the granny2 file at https://forums.eveonline.com/t/triexporter-still-works/59230/2 but still unable to export. Also, double clicking on a model causes TriExporter to crash.

yeah, same issue for me

**Comment by NSF-BKoster** [comment]
Yeah same issue, and i think this application is dead, since no response this entire year.
